Published NPM Version Missing Files

The published version of this package is missing the `src` directory, which causes the following exception when you attempt to call `require("git-changed-files")`:

```
Error: Cannot find module './src/fetchGitStatus'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
```

--- a/index.js
+++ b/index.js
@@ -1,5 +1,3 @@
-const chalk = require('chalk');
-
 const {
   fetchGitStatus
 } = require('./src/fetchGitStatus');

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "git-changed-files",
-  "version": "0.1.0",
+  "version": "0.1.1",
   "description": "Get the change between any of your branch and current branch of a `git` repository",
   "license": "MIT",
   "repository": "https://github.com/kandhavivekraj/git-changed-files",
@@ -12,7 +12,10 @@
     "node": ">=6"
   },
   "files": [
-    "index.js"
+    "index.js",
+    "src/fetchGitStatus.js",
+    "src/filterFiles.js",
+    "src/formatOutput.js"
   ],
   "devDependencies": {
     "eslint": "^5.9.0",

